spec/features/venues/create_new_venue_spec.rb

```diff
@@ -20,5 +20,7 @@
     expect(page).to have_content(venue['website'])
     expect(page).to have_content(venue['address'])
     expect(page).to have_content(t_is_accessible_string)
+
+    expect(page).to have_content('Venue saved successfully')
   end
```

Comment:
This is only testing the happy path. You should test that your flash messages are showing with an unhappy path test or two